manifests/init.pp

```diff
@@ -127,22 +147,26 @@
 
   # Install default instance
   redis::instance { 'redis-default':
```

Comment:
Looks like the `redis::instance` class wasn't updated to accept these parameters, so you'll need to add them there as well.

## Test Case

With node definition:

```puppet
# Vagrant node def
node 'ubuntu2.vagrant.local' {
  class { '::redis':
    redis_repl_backlog_size => "backlog-size",
    redis_slave_output_buffer_hard_limit => "hard-limit",
    redis_slave_output_buffer_soft_limit => "soft-limit",
    redis_slave_output_buffer_soft_limit_max_interval => "interval"
  }
}
```

Error:

```
Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Invalid parameter redis_repl_backlog_size on Redis::Instance[redis-default] at /etc/puppet/modules/redis/manifests/init.pp:170 on node ubuntu2.vagrant.local
Warning: Not using cache on failed catalog
Error: Could not retrieve catalog; skipping run
```